docs/fundamentals/diagnostics/runtime-tiered-compilation-events.md

```diff
@@ -0,0 +1,117 @@
+---
+title: "Tiered compilation runtime events"
+description: See .NET runtime tiered compilation events that collect diagnostic information about tiered compilation in .NET Core.
```

Comment:
```suggestion
description: Learn about the .NET runtime tiered compilation events that collect diagnostic information about tiered compilation in .NET.
```

Reply:
Made a similar change in a few other files.